src/components/AnimatedSignal/AnimatedSignal.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Animated from 'react-native-reanimated';
+import Svg, { Path, Rect } from 'react-native-svg';
+import { Colors, Metrics } from '../../theme';
+import { useAnimatedSignal } from './hooks';
+import type { AnimatedSignalProps } from './types';
+
+const AnimatedSignal = ({
+  wifiArea = 300,
+  numberOfArcs = 4,
+  color = Colors.mediumAquamarine,
+}: Partial<AnimatedSignalProps>) => {
+  const AnimatedPath = Animated.createAnimatedComponent(Path);
```

Comment:
@het-a-simform AnimatedPath should be defined out of the component 
Move following out with imports and a line break
 const AnimatedPath = Animated.createAnimatedComponent(Path);

Reply:
Resolved